dev/tools/localization/bin/gen_l10n.dart

```diff
@@ -72,6 +72,21 @@ void main(List<String> arguments) {
       'Alternatively, see the `header` option to pass in a string '
       'for a simpler header.'
   );
+  parser.addFlag(
+    'use-deferred-loading',
+    defaultsTo: false,
+    help: 'Whether to generate the Dart localization file with locales imported'
+      ' as deferred, to allow for lazy loading of localizations in Flutter web.'
```

Comment:
nit: Can specify which files are lazily loaded:
```suggestion
    help: 'Whether to generate the Dart localization file with locales imported'
      ' as deferred, allowing for lazy loading of each locale in Flutter web.'
```